cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -23,6 +26,95 @@
 import re
 import uuid
 
+
+class GCPResourceType(object):
+
+    def __init__(self, platform, resource_type):
+        self.platform = platform
+        self.resource_type = resource_type
+
+    @property
+    def kind(self):
+        return '%s#%s' % (self.platform, self.resource_type)
+
+    @property
+    def url_field_name(self):
+        return '%ss' % self.resource_type
```

Comment:
Since it was not too difficult, I decided to directly use the resource descriptions coming from googleapis.com instead of using heuristics like this. Please take another look at the new implementation.